src/include/stir/ProjData.h

```diff
@@ -151,6 +152,10 @@ class ProjData : public ExamData
   //! Set sinogram
   virtual Succeeded 
     set_sinogram(const Sinogram<float>&) = 0;
+  //! Get Bin value
+   virtual float get_bin_value(const int view_pos, const int segment_pos, const int axial_pos, const int tang_pos) const = 0;
+  //! Get Bin value
+  virtual float get_bin_value(const Bin& this_bin) const = 0;
```

Comment:
Now they don't make much sense, as I removed the bin-wise operations. But I will keep them for future reference.